app/controllers/doorkeeper/authorizations_controller.rb

```diff
@@ -84,7 +80,13 @@ def strategy
     end
 
     def authorize_response
-      @authorize_response ||= strategy.authorize
+      @authorize_response ||= begin
+        valid = pre_auth.authorizable?
```

Comment:
Let's rename it to `authorizable` to be closer to implementation and the essentials of this method. What do you think?

Reply:
Good call -- I'll change it.